apps/drupal-default/particle_theme/templates/block/block--utc-highlights-block--default.html.twig

```diff
@@ -111,42 +87,15 @@
 					<hr class="border-t-3 border-utc-new-gold-500 mt-3 mb-6 ml-8 w-20" />
 				{% endif %}
 					<ul class="flex flex-row flex-wrap list-none pl-0 mb-0">
```

Comment:
The loop below is very similar to how I did this in the button group...thinking accessing variables might be similar to the "Link" field type. However, this isn't working for me. I tried a few different variations and then backed up from the loop to explore drilling down to the items without a loop. No dice so far after a good bit of research and testing. When I started getting in to the module files/php to determine if something needed reworking there I figured I should check with you first in case the answer was an obvious one! Very possible. ;)

So...any ideas?

If not, I can keep researching.

If we can get them out individually, I think the rewrite to use this widget here will be pretty straightforward from there.

Reply:
Oh, and to be clear "field highlighted icon" is the name of the UTCWidget field.

Reply:
Because of the for loop i probably would not leave those variables on the top of the page as variables but i used that for debugging. Also field_bas is your field_highlighted_icon.
Lastly because we are getting the information in such a granular form.
<img width="586" alt="Screen Shot 2021-10-18 at 8 38 25 PM" src="https://user-images.githubusercontent.com/39039024/137825075-76b0d6ee-e47f-4e3b-93cb-6b9e70fad21a.png">